net/tcp/tcp.h

```diff
@@ -233,7 +233,9 @@ struct tcp_conn_s
   uint16_t tx_unacked;    /* Number bytes sent but not yet ACKed */
 #endif
   uint16_t flags;         /* Flags of TCP-specific options */
-
+#ifdef CONFIG_NET_SOLINGER
+  sclock_t ltimeout;      /* Linger timeout expiration */
```

Comment:
Let's reuse s_linger as the absolute timeout?

Reply:
Done